src/test/java/dev/morling/jfrunit/JfrUnitTest.java

```diff
@@ -70,4 +70,34 @@ public void shouldHaveGcAndSleepEventsWithDefaultConfiguration() throws Exceptio
 
         assertThat(allocated).isGreaterThan(0);
     }
+
+    @Test
```

Comment:
I think there should be another set of test methods: for an event type which has stack traces disabled by default. I believe there'd be an issue if the `@EnableEvent` annotation wouldn't contain the `stackTrace()` attribute, in which case the stack trace would be enabled by means of the default value of that attribute. It's that why I had foreseen that third enum member, `DEFAULT.

Reply:
Agreed on above scenario where events can have stack traces disabled by default and thus requiring the third enum member, `DEFAULT. I'll add test cases for the same.